articles/operator-nexus/howto-cluster-metrics-configuration-management.md

```diff
@@ -126,3 +174,18 @@ az rest -m delete -u "https://management.azure.com/subscriptions/${SUBSCRIPTION}
 ```
 
 Specifying `--debug` in REST API will result in the tracking operation status in the returned command output. This operation status can be queried to monitor the progress of the operation. See: [How-to track asynchronous operations](howto-track-async-operations-cli.md).
+
+### Parameters for cluster metrics configuration operations
+
+| Parameter name                        | Description                                                                                                           |
+| --------------------------------------| --------------------------------------------------------------------------------------------------------------------- |
+| CLUSTER                               | Resource Name of the Cluster                                                                                          |
```

Comment:
go over all descriptions and clean them up:
- Create should be lower cased.
- Cluster should be capitalized everywhere